src/engraving/libmscore/cmd.cpp

```diff
@@ -27,6 +27,8 @@
 
 #include <assert.h>
 
+#include "engravingitem.h"
+#include "engravingobject.h"
```

Comment:
These includes are also not necessary. They happen automatically by the other includes. 

(You're right that in general these "transitive includes" are not always great, but in the case of these particular headers, I think we should allow it.)